src/ReactNativeZoomableView.tsx

```diff
@@ -42,6 +42,8 @@ const initialState: ReactNativeZoomableViewState = {
   originalHeight: 0,
   originalPageX: 0,
   originalPageY: 0,
+  originalX: 0,
+  originalY: 0,
```

Comment:
This looks right but I'm a little confused about the `original` prefix on these variables

Reply:
They represent the "original" layout of the zoomed and panned view